roosevelt.js

```diff
@@ -138,6 +129,15 @@ module.exports = function(params) {
       require('./lib/jsCompiler')(app, mapRoutes);
     }
 
+     // initialize HTML validator
+    validateHTML();
+
+    function validateHTML() {
+      require('./lib/htmlValidator')(app, preprocessCss);
```

Comment:
yeah - just to be super clear: the goal here is to get the css/js parsing stuff to happen first, *then* begin starting the HTML validator so we change the order of the console output